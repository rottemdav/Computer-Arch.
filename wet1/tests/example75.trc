8 5 27 1 local_history global_tables using_share_mid
0x7a8 T 0x46d375c
0x6d55ab0 N 0x7fc0d750
0xb654480 T 0xe200
0x1f04 T 0x52c
0x7150 N 0xfa41f8e0
0xdf4 N 0x57abd0
0xdd40 T 0x569b0cc
0x666dbe30 N 0x0716ca0
0x46bec T 0x3520
0xd40 N 0x850
0xbf0 T 0xc2b70
0x1f45d4 N 0x2cd018e4
0xcd68 N 0x9cbe0
0xbf0 N 0xc2b70
0x6d55ab0 T 0x7fc0d750
0xa9d0cc T 0xeee8
0x678bcd4 N 0x19a0
0xdd40 T 0x569b0cc
0x1f04 N 0x52c
0xd5cb774 T 0x5c14094
0x8368 T 0x9605d4
0xb654480 T 0xe200
0x1f45d4 T 0x2cd018e4
0x7d70397c T 0x51f10
0x8368 N 0x9605d4
0x666dbe30 T 0x0716ca0
0xa9d0cc T 0xeee8
0xcaa9a4c T 0x1bc
0x7a8 N 0x46d375c
0xf427504 T 0x7b0
0xcaa9a4c N 0x1bc
0xa6c04 T 0x714c
0x1646fb0 N 0xe2c
0xf427504 T 0x7b0
0xd5cb774 T 0x5c14094
0x42c61844 N 0xb7c9408
0x7d70397c N 0x51f10
0x9c25c T 0x3e8
0x46bec T 0x3520
0xd40 T 0x850
0x4988ab4 N 0x09374
0xcaa9a4c N 0x1bc
0x46bec N 0x3520
0x7150 T 0xfa41f8e0
0xdf4 N 0x57abd0
0x666dbe30 N 0x0716ca0
0xf352b6cc T 0xf60bd0
0xf352b6cc N 0xf60bd0
0xd40 T 0x850
0x7f4c N 0x84e07c
0x25c6bf4 T 0x6be0
0x4988ab4 N 0x09374
0x678bcd4 N 0x19a0
0xdf4 T 0x57abd0
0x47aac T 0x9f2da090
0x42c61844 T 0xb7c9408
0x9c25c T 0x3e8
0x25c6bf4 N 0x6be0
0x7f4c N 0x84e07c
0xc6738 T 0xb6794
0x1646fb0 N 0xe2c
0x1646fb0 T 0xe2c
0x25c6bf4 N 0x6be0
0x666dbe30 N 0x0716ca0
0x9c25c T 0x3e8
0x7d70397c N 0x51f10
0x1646fb0 N 0xe2c
0x7150 N 0xfa41f8e0
0x6a64 T 0xe6d21104
0x1f04 N 0x52c
0x8ee4c4 N 0xedfe4578
0xcaa9a4c T 0x1bc
0x678bcd4 T 0x19a0
0xdd40 N 0x569b0cc
0xa9d0cc T 0xeee8
0x2ec N 0xae638
0x47aac T 0x9f2da090
0xbf0 T 0xc2b70
0xff8c0 N 0x830
0x2b6b4 N 0x4840
0x4988ab4 T 0x09374
0x1f45d4 N 0x2cd018e4
0xf352b6cc N 0xf60bd0
0xfff9fc4 T 0x856cc
0x9c25c N 0x3e8
0xa9d0cc T 0xeee8
0xc6738 T 0xb6794
0x46bec T 0x3520
0xcaa9a4c N 0x1bc
0x6a64 N 0xe6d21104
0xcd68 N 0x9cbe0
0x7150 N 0xfa41f8e0
0xfff9fc4 N 0x856cc
0xcaa9a4c N 0x1bc
0x6a64 N 0xe6d21104
0xcaa9a4c T 0x1bc
0x6a64 T 0xe6d21104
0xbf0 N 0xc2b70
0x7150 T 0xfa41f8e0
0x4988ab4 N 0x09374
0xff8c0 N 0x830
0xff8c0 N 0x830
0x8ee4c4 T 0xedfe4578
0x7f4c N 0x84e07c