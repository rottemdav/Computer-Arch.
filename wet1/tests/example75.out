0x7a8 N 0x7ac
0x6d55ab0 N 0x6d55ab4
0xb654480 N 0xb654484
0x1f04 N 0x1f08
0x7150 N 0x7154
0xdf4 N 0xdf8
0xdd40 N 0xdd44
0x666dbe30 N 0x666dbe34
0x46bec N 0x46bf0
0xd40 N 0xd44
0xbf0 N 0xbf4
0x1f45d4 N 0x1f45d8
0xcd68 N 0xcd6c
0xbf0 N 0xbf4
0x6d55ab0 N 0x6d55ab4
0xa9d0cc N 0xa9d0d0
0x678bcd4 N 0x678bcd8
0xdd40 N 0xdd44
0x1f04 N 0x1f08
0xd5cb774 N 0xd5cb778
0x8368 N 0x836c
0xb654480 N 0xb654484
0x1f45d4 N 0x1f45d8
0x7d70397c N 0x7d703980
0x8368 N 0x836c
0x666dbe30 N 0x666dbe34
0xa9d0cc N 0xa9d0d0
0xcaa9a4c N 0xcaa9a50
0x7a8 N 0x7ac
0xf427504 N 0xf427508
0xcaa9a4c N 0xcaa9a50
0xa6c04 N 0xa6c08
0x1646fb0 N 0x1646fb4
0xf427504 N 0xf427508
0xd5cb774 N 0xd5cb778
0x42c61844 N 0x42c61848
0x7d70397c N 0x7d703980
0x9c25c N 0x9c260
0x46bec N 0x46bf0
0xd40 N 0xd44
0x4988ab4 N 0x4988ab8
0xcaa9a4c N 0xcaa9a50
0x46bec N 0x46bf0
0x7150 N 0x7154
0xdf4 N 0xdf8
0x666dbe30 N 0x666dbe34
0xf352b6cc N 0xf352b6d0
0xf352b6cc N 0xf352b6d0
0xd40 N 0xd44
0x7f4c N 0x7f50
0x25c6bf4 N 0x25c6bf8
0x4988ab4 N 0x4988ab8
0x678bcd4 N 0x678bcd8
0xdf4 N 0xdf8
0x47aac N 0x47ab0
0x42c61844 N 0x42c61848
0x9c25c T 0x3e8
0x25c6bf4 N 0x25c6bf8
0x7f4c N 0x7f50
0xc6738 N 0xc673c
0x1646fb0 N 0x1646fb4
0x1646fb0 N 0x1646fb4
0x25c6bf4 T 0x6be0
0x666dbe30 N 0x666dbe34
0x9c25c T 0x3e8
0x7d70397c N 0x7d703980
0x1646fb0 N 0x1646fb4
0x7150 N 0x7154
0x6a64 N 0x6a68
0x1f04 N 0x1f08
0x8ee4c4 N 0x8ee4c8
0xcaa9a4c N 0xcaa9a50
0x678bcd4 N 0x678bcd8
0xdd40 N 0xdd44
0xa9d0cc N 0xa9d0d0
0x2ec N 0x2f0
0x47aac N 0x47ab0
0xbf0 N 0xbf4
0xff8c0 N 0xff8c4
0x2b6b4 N 0x2b6b8
0x4988ab4 N 0x4988ab8
0x1f45d4 N 0x1f45d8
0xf352b6cc N 0xf352b6d0
0xfff9fc4 N 0xfff9fc8
0x9c25c N 0x9c260
0xa9d0cc N 0xa9d0d0
0xc6738 N 0xc673c
0x46bec N 0x46bf0
0xcaa9a4c N 0xcaa9a50
0x6a64 N 0x6a68
0xcd68 N 0xcd6c
0x7150 N 0x7154
0xfff9fc4 N 0xfff9fc8
0xcaa9a4c T 0x1bc
0x6a64 N 0x6a68
0xcaa9a4c N 0xcaa9a50
0x6a64 N 0x6a68
0xbf0 N 0xbf4
0x7150 N 0x7154
0x4988ab4 N 0x4988ab8
0xff8c0 N 0xff8c4
0xff8c0 N 0xff8c4
0x8ee4c4 N 0x8ee4c8
0x7f4c N 0x7f50
flush_num: 50, br_num: 104, size: 568b
